### PR TITLE
Add project guidelines and issues/prs templates.

### PR DESCRIPTION
**Pull request by caiovfernandes**
This pull request push project guidelines and templates files to improve repository community health.

## Types of changes
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)

## Checklist

- [x] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report_form.yml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    id: logs
+    attributes:
+      label: Relevant log output
+      description: Please copy and paste any relevant log output. This will be automatically formatted into code, so no need for backticks.
+      render: shell
+  - type: checkboxes
+    id: terms
+    attributes:
+      label: Code of Conduct
+      description: By submitting this issue, you agree to follow our [Code of Conduct](./CODE_OF_CONDUCT.md)
```
Comment (arthurbdiniz) [inline .github/ISSUE_TEMPLATE/bug_report_form.yml:74]: ../blob/master/CODE_OF_CONDUCT.md

File: .github/ISSUE_TEMPLATE/bug_report_form.yml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+      label: Expected behavior
+      description: Describe the expected behavior
+      placeholder: The expected behavior is ...
+    validations:
+      required: false
+  - type: textarea
+    id: actual-behavior
+    attributes:
+      label: Actual behavior
+      description: Describe the actual behavior
```
Comment (arthurbdiniz) [inline .github/ISSUE_TEMPLATE/bug_report_form.yml:49]: Describe the actual behavior.

File: .github/ISSUE_TEMPLATE/bug_report_form.yml [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+      placeholder: 1. Step 1 ...
+                   2. Step 2 ...
+                   3. Step 3 ...
+    validations:
+      required: true
+  - type: textarea
+    id: expected-behavior
+    attributes:
+      label: Expected behavior
+      description: Describe the expected behavior
```
Comment (arthurbdiniz) [inline .github/ISSUE_TEMPLATE/bug_report_form.yml:41]: Describe the expected behavior.

File: .github/ISSUE_TEMPLATE/feature_request_form.yml [hunk 4]
```diff
@@ -0,0 +1,50 @@
+name: Feature Request
+description: Suggest an idea for this project
+title: "[Feature-Request]: "
```
Comment (arthurbdiniz) [inline .github/ISSUE_TEMPLATE/feature_request_form.yml:3]: Remove title

File: .github/ISSUE_TEMPLATE/feature_request_form.yml [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+    attributes:
+      label: Motivation
+      description: Describe the motivation behind this feature request.
+    validations:
+      required: true
+  - type: textarea
+    id: alternatives
+    attributes:
+      label: Alternatives
+      description: Describe the alternatives you've considered
```
Comment (arthurbdiniz) [inline .github/ISSUE_TEMPLATE/feature_request_form.yml:36]: Describe the alternatives you've considered.

File: .github/ISSUE_TEMPLATE/feature_request_form.yml [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+      required: false
+  - type: textarea
+    id: additional-information
+    attributes:
+      label: Additional Context
+  - type: checkboxes
+    id: terms
+    attributes:
+      label: Code of Conduct
+      description: By submitting this issue, you agree to follow our [Code of Conduct](../../CODE_OF_CONDUCT.md)
```
Comment (arthurbdiniz) [inline .github/ISSUE_TEMPLATE/feature_request_form.yml:47]: ../blob/master/CODE_OF_CONDUCT.md

File: .github/ISSUE_TEMPLATE/issue_template.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
+
```
Comment (arthurbdiniz) [inline .github/ISSUE_TEMPLATE/issue_template.md:1]: do we need this file?

File: .github/pull_request_template.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+- [ ] Bugfix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Documentation Update (if none of the other choices apply)
+
+## Checklist
+
+_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._
+
+- [ ] I have read the [CONTRIBUTING](https://github.com/appium/appium/blob/master/CONTRIBUTING.md) doc
```
Comment (arthurbdiniz) [inline .github/pull_request_template.md:18]: I have read the CONTRIBUTING doc.

File: .github/pull_request_template.md [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+- [ ] I have added necessary documentation (if appropriate)
```
Comment (arthurbdiniz) [inline .github/pull_request_template.md:19]: I have added necessary documentation (if appropriate).

File: .github/pull_request_template.md [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+- [ ] Any dependent changes have been merged and published in downstream modules
```
Comment (arthurbdiniz) [inline .github/pull_request_template.md:20]: Any dependent changes have been merged and published in downstream modules.

File: CONTRIBUTING.MD [hunk 11]
```diff
@@ -0,0 +1,76 @@
...
+
+* **Use a clear and descriptive title** for the issue to identify the problem.
+* **Describe the exact steps which reproduce the problem** in as many details as possible. For example, start by explaining how you started the module, e.g. which command exactly you used in the terminal.
+* **Provide specific examples to demonstrate the steps**. Include links to files or GitHub projects, or copy/pasteable snippets, which you use in those examples. If you're providing snippets in the issue, use [Markdown code blocks](https://docs.github.com/pt/github/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks).
+* **Describe the behavior you observed after following the steps** and point out what exactly is the problem with that behavior.
+* **If the problem wasn't triggered by a specific action**, describe what you were doing before the problem happened and share more information using the guidelines below.
+
+Provide more context by answering these questions:
+
+* **Did the problem start happening recently** (e.g. after updating to a new version of Terraform) or was this always a problem?
```
Comment (arthurbdiniz) [inline CONTRIBUTING.MD:29]: e.g. after updating to a new version of Docker

File: CONTRIBUTING.MD [hunk 12]
```diff
@@ -0,0 +1,76 @@
...
+* If the problem started happening recently, **can you reproduce the problem in an older version of Terraform?** What's the most recent version in which the problem doesn't happen? You can download older versions of Terraform from [the releases page](https://github.com/hashicorp/terraform/releases).
```
Comment (arthurbdiniz) [inline CONTRIBUTING.MD:30]: can you reproduce the problem in an older version of Docker?

Comment (arthurbdiniz) [reply CONTRIBUTING.MD:30]: You can download older versions of Terraform from [the releases page](https://github.com/hashicorp/terraform/releases)

Point to Docker release.

File: CONTRIBUTING.MD [hunk 13]
```diff
@@ -0,0 +1,76 @@
...
+* **Can you reliably reproduce the issue?** If not, provide details about how often the problem happens and under which conditions it normally happens.
+* If the problem is related to working with files (e.g. opening and editing files), **does the problem happen for all files and projects or only some?** Does the problem happen only when working with local or remote files (e.g. on network drives), with files of a specific type (e.g. only JavaScript or Python files), with large files or files with very long lines, or with files in a specific encoding? Is there anything else special about the files you are using?
+### Suggesting Enhancements
+
+This section guides you through submitting an enhancement suggestion for DNX modules, including completely new features and minor improvements to existing functionality. Following these guidelines helps maintainers and the community understand your suggestion and find related suggestions.
```
Comment (arthurbdiniz) [inline CONTRIBUTING.MD:35]: From:
```
This section guides you through submitting an enhancement suggestion for DNX modules
```
To:
```
This section guides you through submitting an enhancement suggestion for DNX
```

**Review by arthurbdiniz — CHANGES_REQUESTED** [review]
Found some changes.

File: .github/ISSUE_TEMPLATE/issue_template.md (hunk 7 above)
Comment (caiovfernandes) [reply .github/ISSUE_TEMPLATE/issue_template.md:1]: Removed ✅

File: .github/ISSUE_TEMPLATE/bug_report_form.yml (hunk 1 above)
Comment (caiovfernandes) [reply .github/ISSUE_TEMPLATE/bug_report_form.yml:74]: Done  ✅

File: .github/ISSUE_TEMPLATE/bug_report_form.yml (hunk 3 above)
Comment (caiovfernandes) [reply .github/ISSUE_TEMPLATE/bug_report_form.yml:41]: Done  ✅

File: .github/ISSUE_TEMPLATE/bug_report_form.yml [hunk 14]
```diff
@@ -0,0 +1,77 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
```
[the remark this answers is not in the document]
Comment (caiovfernandes) [reply .github/ISSUE_TEMPLATE/bug_report_form.yml:3]: Done  ✅

File: .github/ISSUE_TEMPLATE/feature_request_form.yml (hunk 4 above)
Comment (caiovfernandes) [reply .github/ISSUE_TEMPLATE/feature_request_form.yml:3]: Done  ✅

File: .github/ISSUE_TEMPLATE/feature_request_form.yml (hunk 5 above)
Comment (caiovfernandes) [reply .github/ISSUE_TEMPLATE/feature_request_form.yml:36]: Done  ✅

File: .github/ISSUE_TEMPLATE/feature_request_form.yml (hunk 6 above)
Comment (caiovfernandes) [reply .github/ISSUE_TEMPLATE/feature_request_form.yml:47]: Done  ✅

File: .github/pull_request_template.md (hunk 8 above)
Comment (caiovfernandes) [reply .github/pull_request_template.md:18]: Done  ✅

File: .github/pull_request_template.md (hunk 9 above)
Comment (caiovfernandes) [reply .github/pull_request_template.md:19]: Done  ✅

File: .github/pull_request_template.md (hunk 10 above)
Comment (caiovfernandes) [reply .github/pull_request_template.md:20]: Done  ✅

File: CONTRIBUTING.MD (hunk 11 above)
Comment (caiovfernandes) [reply CONTRIBUTING.MD:29]: Done  ✅

File: CONTRIBUTING.MD (hunk 12 above)
Comment (caiovfernandes) [reply CONTRIBUTING.MD:30]: Done  ✅

File: CONTRIBUTING.MD (hunk 13 above)
Comment (caiovfernandes) [reply CONTRIBUTING.MD:35]: Done  ✅

File: .github/ISSUE_TEMPLATE/issue_template.md (hunk 7 above)
Comment (arthurbdiniz) [reply .github/ISSUE_TEMPLATE/issue_template.md:1]: This file is still commited

File: .github/pull_request_template.md (hunk 8 above)
Comment (arthurbdiniz) [reply .github/pull_request_template.md]: Contributing is pointing to external link https://github.com/appium/appium/blob/master/CONTRIBUTING.md

File: .github/ISSUE_TEMPLATE/issue_template.md (hunk 7 above)
Comment (caiovfernandes) [reply .github/ISSUE_TEMPLATE/issue_template.md:1]: Done ✅

File: .github/pull_request_template.md (hunk 8 above)
Comment (caiovfernandes) [reply .github/pull_request_template.md]: Done ✅

**Review by arthurbdiniz — APPROVED** [review]

**Pull request merged**